fiftyone/utils/fcclip.py

```
@@ -0,0 +1,172 @@
+
+
+class FCCLIPOutputProcessor(fout.OutputProcessor):
+    """Converts FC-CLIP panoptic segmentation outputs to
+    :class:`fiftyone.core.labels.Detections`.
+    """
+
+    def __init__(self, classes=None, **kwargs):
+        super().__init__(classes=classes, **kwargs)
+        self.classes = classes
+
+    def __call__(
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__call__`

(ANN204)